src/components/Table/utils.js

```diff
@@ -18,12 +21,33 @@ export const getSortDirection = (isActive, currentSort) => {
   return currentSort === ASCENDING ? DESCENDING : ASCENDING;
 };
 
-export const ascendingSort = curry(
-  (i, a, b) => getSortByValue(a[i]) > getSortByValue(b[i])
-);
-export const descendingSort = curry(
-  (i, a, b) => getSortByValue(a[i]) < getSortByValue(b[i])
-);
+export const ascendingSort = curry((i, a, b) => {
+  const first = getSortByValue(a[i]);
+  const second = getSortByValue(b[i]);
+
+  if (first < second) {
+    return -1;
+  }
+  if (first > second) {
+    return 1;
+  }
+
+  return 0;
```

Comment:
A neater way of doing this would be:

```js
(i, a, b) => getSortByValue(a[i]) - getSortByValue(b[i])
```

Reply:
You sure? These aren't only numbers. They can also be strings

Reply:
I assumed they were numbers Can you compare strings with `>`? 😮

Reply:
Yeah. they get compared with `length` doing so